setup.py

```diff
@@ -232,7 +232,15 @@ def main(argv):
             packages=find_packages(exclude=[
                 'telethon_*', 'tests*'
             ]),
-            install_requires=['pyaes', 'rsa'],
+
+            # List run-time dependencies here. These will be installed by pip when your
+            # project is installed. For an analysis of "install_requires" vs pip's
+            # requirements files see:
```

Comment:
The link is fine, but I think this is unnecessary. We're not here to teach others how to write a `setup.py` file.